lib/geordi/gitpt.rb

```diff
@@ -57,19 +64,26 @@ def load_projects
   end
 
   def read_project_ids
-    file_path = PROJECT_IDS_FILE_NAME
+    project_ids = settings.pivotal_tracker_project_id
 
-    if File.exist?(file_path)
-      project_ids = File.read('.pt_project_id').split(/[\s]+/).map(&:to_i)
+    file_path = PROJECT_IDS_FILE_NAME
+    if !project_ids and File.exist?(file_path)
+      project_ids = File.read('.pt_project_id')
+      Geordi::Interaction.warn "The usage of the .pt_project_id file is deprecated."
+      Geordi::Interaction.note Util.strip_heredoc(<<-INSTRUCTIONS)
+          Please remove this file from your project and add or extend the .geordi.yml file with the following content:
+            pivotal_tracker_project_id: #{project_ids}
+      INSTRUCTIONS
     end
 
-    if project_ids && (project_ids.size > 0)
-      project_ids
+    if project_ids && (project_ids.to_s.length > 0)
+      project_ids.to_s.split(/[\s]+/).map(&:to_i)
     else
-      Geordi::Interaction.warn "Sorry, I could not find a project ID in #{file_path} :("
+      Geordi::Interaction.warn "Sorry, I could not find a project ID in .geordi.yml :("
       puts
 
-      puts "Please put at least one Pivotal Tracker project id into #{file_path} in this directory."
+      puts "Please put at least one Pivotal Tracker project id into the .geordi.yml file in this directory, e.g:\n"
+      puts "pivotal_tracker_project_id: 123456\n"
       puts 'You may add multiple IDs, separated using white space.'
       exit 1
     end
```

Comment:
Please move this whole function to `Geordi::Settings`. It should always return an array of project ids, and may serialize it as Yaml array OR white-space separated string.